Volume/Views/MainView/HomeView.swift

```diff
@@ -71,6 +76,17 @@ struct HomeView: View {
     }
 
     // MARK: Sections
+    
+    private var searchBar: some View {
+        NavigationLink {
+            SearchView()
+        } label: {
+            SearchBar(searchState: $viewModel.searchState, searchText: $viewModel.searchText)
+                .disabled(true)
+                .padding(EdgeInsets(top: Constants.navBarVerticalPadding, leading: Constants.listHorizontalPadding, bottom: Constants.rowVerticalPadding,
+                    trailing: Constants.listHorizontalPadding))
```

Comment:
Having the same search bar here as the one on `SearchView` might be a little awkward for some users, as they expect when tapping on it to stay on the same page and begin searching. There's a cognitive conflict with the search bar you are searching in being different from the search bar that you tapped, as well as the fact that they are on different screens. i.e. "why are there two"?

Perhaps you could redesign the search system to replace `listContent` on `HomeView` with the search dropdown content in `SearchView` so you don't have to navigate to a new view in order to search. `SearchView` composes the parts in a `VStack` anyway.